src/backend/marsha/core/api.py

```diff
@@ -309,6 +310,8 @@ def update_live_state(self, request, pk=None):
         video.live_info = live_info
         video.save()
 
+        live_status_changed.send(sender=self.__class__, video=video)
```

Comment:
I don't see the interest of going through a signal for this versus calling the method directly.
Signals add complexity and are hard to follow when you read the code.

Something maybe more straightforward is, before doing video.save(), trying to delete the stack if the new state requires it:
```python
if video.live_state == STOPPED:
    delete_aws_element_stack(video)

video.save()
```

Reply:
Ok I will remove the signal